ql/math/optimization/simulatedannealing.hpp

```diff
@@ -42,6 +42,7 @@ namespace QuantLib {
     */
 
     template <class RNG = MersenneTwisterUniformRng>
+    //! Simulated Annealing
```

Comment:
Thanks! For consistency with the rest of the code, I'd move this above the `template` line.

Reply:
Thanks for the prompt reply. Change performed.